Sources/Core/File.swift

```diff
@@ -13,7 +13,13 @@ public struct File: Codable {
 
     /// The file extension, if it has one.
     public var ext: String? {
-        return filename.split(separator: ".").last.map(String.init)
+        let splitted = filename.split(separator: ".")
```

Comment:
"to split" is irregular and is still "split" in the past tense. Another name for this could be `parts`.